### ISSUE TITLE
Usage of classNameFilter

### ISSUE THREAD

**Issue by michalvvv** [angular-ui/bootstrap#4404]
Until version without using `ngAnimate`, it was possible to use `$animateProvider.classNameFilter(/animate/)` (this line increased page speed noticeably).
It would be great if it was still available in actual code.

**Comment by wesleycho** [comment]
Can you explain clearly what you mean? Do you mean until `ngAnimate` was used? Nothing is stopping you from using `$animateProvider.classNameFilter` - this sounds more like a user error than anything else.

**Comment by michalvvv** [comment]
Ok, I see now it wasn't clear message. Sorry.

Previously you didn't use ngAnimate, but my app did. So I was able to use `$animateProvider.classNameFilter(/animate/);` line and everything animated properly.

Now I'm migrating to a newer Angular version. After changing library files, accordions and collapse stopped working. I found that above code line was the problem: http://plnkr.co/edit/uGjQzKCy6hrj0V8MVBxD?p=preview (comment/uncomment line 4 in example.js).

Is it your library's problem or angular-animate's problem that it doesn't add `/animate/` while running `animateCss` or should it work in this way?
